bin/fresh-node10

```diff
@@ -53,6 +53,10 @@ if [ -e "$mountsrc/.git" ]; then
 	welcometxt_suffix+="#        $mountdest/.git ➟ $mountsrc/.git (read-only)\n"
 fi
 if [ -n "$allow_env_mw" ]; then
+	# Source the local .env file if it exists (T246586)
+	if [ -f .env ]; then
+		source .env
```

Comment:
I assume this would contain things like `FOO=1` and not actual (intentionally) running shell commands like `export` or `set`.

But, afaik such assingment cannot be read with `source` as they don't seem to persist?

Hm.. 

```shell
$ env | grep FOO
(empty)

$ echo FOO=1 > .env

$ source .env
$ env | grep FOO
(still empty)

$ (source .env && ( env | grep FOO)) 
(also empty)
```